config/core-site.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+
+<!-- WASB (Azure Storage) Configuration  -->
+<!-- <property>
+  <name>fs.AbstractFileSystem.wasb.Impl</name>
+  <value>org.apache.hadoop.fs.azure.Wasb</value>
+</property>
+<property>
+  <name>fs.azure.account.key.$storage_account_name.blob.$storage_account_suffix</name>
```

Comment:
`$storage_account_name` - looks like a variable. But they should be hardcoding it right here. maybe it should be `_STORAGE_ACCOUNT_NAME_` or something